packs/rackspace/sensors/README.md

```diff
@@ -0,0 +1,4 @@
+# sensors
```

Comment:
We can remove those placeholder readmes in the resource directories. Just need to have README in top-level pack directory.